### DISCUSSION TITLE
Supported players

### DISCUSSION THREAD

**Discussion by Suriya2210** [bloc97/Anime4K#193] [Q&A] [answered]
So, I want to use the upscaler on any player other than MPV. But I can't find a supported player to use it (I might have missed it somewhere). It would be great if I get a list of players that support the installation of the upscaler.

**Comment by bloc97** [comment, marked as answer]
Any media player that uses [mpv](https://[mpv.io](https://mpv.io/)/) and/or [libplacebo](https://github.com/haasn/libplacebo) behind will work (eg. plex). Unfortunately I do not know about any list of media players. As a last resort you can also use [Magpie](https://github.com/Blinue/Magpie) to upscale the window of the media player to full screen (Magpie supports Anime4K shaders and other shaders too).

**Comment by Tama47** [comment]
So far, the only 3 players I've used with and worked are mpv, Plex, and IINA (Mac).

**Comment by monarc99** [comment]
list of mpv based players:

https://github.com/stax76/awesome-mpv#media-player

**Comment by iXeed05** [comment]
How to do this on android? Can you give me instructions?

**Reply by Tama47** [reply]
https://github.com/bloc97/Anime4K/issues/99#issuecomment-897873000

**Reply by iXeed05** [reply]
![Uploading Screenshot_20230507_100339.jpg…]()

**Reply by iXeed05** [reply]
![Uploading Screenshot_20230507_101410.jpg…]()

**Reply by iXeed05** [reply]
Not working

**Reply by Tama47** [reply]
I don’t have an android, so I can’t really test it out. Sorry.
